webservices/utils.py

```diff
@@ -335,3 +340,15 @@ def create_eregs_link(part, section):
     if section:
         url_part_section += '-' + section
     return '/regulations/{}/CURRENT'.format(url_part_section)
+
+def post_to_slack(message, channel):
+    response = requests.post(
+        env.get_credential('SLACK_HOOK'),
```

Comment:
Does the `SLACK_HOOK` environment variable need to be managed outside in the credential service?

Reply:
I think we need to add this to circle- I will do that now

Reply:
done, now we can have things slack us in circle if we want it to